dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -942,14 +962,25 @@ STDAPI MrmGetFilePathFromName(_In_ PCWSTR filename, _Outptr_ PWSTR* filePath)
         {
             // If none of the file exists, will return the file in current path
```

Comment:
>If none of the file exists [](http://example.com/codeflow?start=15&length=26)

Using "If the pending second iteration doesn't find the file either" instead of this would make it easier to understand this comment.